README.md

```diff
@@ -4,32 +4,43 @@ __URLs__
 
 * Name all of the parts of the url that you can remember.  In your own words describe what they do.
 * Name the pieces of the following urls:
-	* `https://www.google.com/`
-	* `https://workbook.galvanize.com/cohorts/41/learning_experiences/367`
-	* `http://locahost:5000/animals/puppies?onlycute=1&size=medium#firstpuppy`
-	* `https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error`
+	* `https://www.google.com/`-->protocol, domain
+	* `https://workbook.galvanize.com/cohorts/41/learning_experiences/367` -->protocol, domain, path
+	* `http://locahost:5000/animals/puppies?onlycute=1&size=medium#firstpuppy` -->protocol, domain, port, path, query string, anchor tag
+	* `https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error` -->protocol, domain, path, anchor tag
 * Can a server use more than 1 port?
+```Yes```
 * Why is https different than http?
+```it is the secure version of http```
 * How does a server interpret the following url's query paramter.  What data structure does it create on the server?
 
 ```
 http://locahost:5000/animals?puppies=fido&puppies=max&puppies=moxie
 ```
 
+```I think that an object is created, more specifically, perhaps, an array```
+
 __HTTP Request/Response__
 
 * Name at least 4 http verbs
+```GET, POST, PUT, DELETE```
 * What is each verb useful for in your own words
+```GET is used to grab or access some resource; POST is used to upload something; PUT is used to modify some data that already exists; and DELETE is used to remove some data.```
 * What does idempotent mean?
+``` It describes an element or value that is unchanged over multiple requests. I think. ```
 * Name the 5 http status code ranges.  What are they used for in general?
+``` 5xx: an error has occurred. 4xx: No error on the server side, but what was requested is not deliverable. 3xx: what was requested exists somewhere else. 2xx: What was requested was delivered in some form. 1xx: What was requested is in the process of being delivered. ```
```

Comment:
👍 
Just to clarify, 500 is a server side error, 400 is generally a user error (request wasn't formed right or was to something invalid)